src/pages/login/login.ts

```diff
@@ -46,15 +43,12 @@ export class LoginPage extends BasePageComponent {
     }
   }
 
-  ionViewDidEnter(): void {
-    this.analytics.setCurrentPage(AnalyticsScreenNames.LOGIN);
```

Comment:
Is this going to be moved somewhere else?

Reply:
Just about to have a discussion with Pam on this one. We can't run the analytics there as we don't have the google analytics id as it's stored remotely and we can't access it until after we have authenticated

Reply:
Spoke to Pam and we will leave it out for now.